python_modules/dagster/dagster/_core/definitions/result.py

```diff
@@ -73,15 +73,13 @@ class MaterializeResult(AssetResult):
 
 @experimental
 class ObserveResult(AssetResult):
-    """An object representing a successful observation of an asset. These can be returned from
```

Comment:
Is it intentional that we're removing the extra context here?

Reply:
Yes. That's not currently public-facing functionality.